modules/nf-core/thermorawfileparser/main.nf

```diff
@@ -0,0 +1,54 @@
+process THERMORAWFILEPARSER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/thermorawfileparser:1.4.3--ha8f3691_0' :
+        'biocontainers/thermorawfileparser:1.4.3--ha8f3691_0' }"
+
+    input:
+    tuple val(meta), path(raw)
+
+    output:
+    tuple val(meta), path("*.{mzML,mfg,parquet}"), emit: spectra
+    path "versions.yml"                          , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def formatSuffixMap = ["0": "mgf", "1": "mzML", "2": "mzML", "3": "parquet"]
+    // Please provide the output format via --format, not -f. If --format not specified, the parser defaults to mzML
+    suffix = formatSuffixMap.get(task.ext.args?.format, "mzML")
```

Comment:
I tested this as I wasn't familiar with the syntax, I got an error. Are you certain this is correct?
```Error
ERROR ~ Error executing process > 'THERMORAWFILEPARSER (test)'       
Caused by: 
 No such variable: format -- Check script '/workspace/modules/./modules/nf-core/thermorawfileparser/tests/../main.nf' at line: 25
``` 
Why not do something similar to bcftools? 
```suggestion
 suffix = args.contains("--format 0") || args.contains("-f 0") ? "mgf.gz" :
            args.contains("--format 1") || args.contains("-f 1") ? "mzML.gz" :
            args.contains("--format 2") || args.contains("-f 2") ? "mzML.gz" :
            args.contains("--format 3") || args.contains("-f 3") ? "parquet.gz" :
            "mzML.gz"
```

Reply:
Ugh, good catch! My bad here. Thanks for the suggestion! Will include it without gz compression